src/plays/markdown-editor/styles.css

```diff
@@ -0,0 +1,26 @@
+/* enter stlyes here */
+*{
+    box-sizing: border-box; 
+  }
+  
+  .play-details{
+    background-color: #724040;
+  }
+  .main{
```

Comment:
This is has got higher chances of getting overridden. Please use a unique class name.